.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "editor.codeActionsOnSave": {
+    "source.sortImports": true
+  },
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "editor.formatOnSave": true
+}
```

Comment:
Copied this from work lol - if you're using VSCode it'll auto-format so we don't have to run the formatter manually as much.